activejob/test/cases/translation_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'helper'
+require 'jobs/translated_hello_job'
+
+class TranslationTest < ActiveSupport::TestCase
+  setup do
+    JobBuffer.clear
+    I18n.available_locales = [:en, :de]
+    @job = TranslatedHelloJob.new('Johannes')
+  end
+
+  teardown do
+    I18n.available_locales = [:en]
+  end
+
+  test 'it performs the job in the given locale' do
+    @job.locale = :de
+    @job.perform_now
+    assert_equal "Johannes says hallo", JobBuffer.last_value
```

Comment:
The expected value doesn't sound very german. Which makes me think the test is faulty (doesn't mean it is though). Am I missing something?